tests/MongoDB.Driver.Tests/Specifications/command-monitoring/tests/find.yml

```diff
@@ -52,7 +52,6 @@ tests:
           $min: { _id: 0 }
           $returnKey: false
           $showDiskLoc: false
-          $snapshot: false
```

Comment:
We need to resync with the tests in the specifications repo, but for now I just want to manually fix these two tests.